frontend/components/siteList/siteList.js

```diff
@@ -48,9 +48,33 @@ const addWebsiteButton = () => (
   <Link
     to="/sites/new"
     role="button"
-    className="usa-button"
+    className="usa-button button-add-website"
     alt="Add a new website"
   >
+    <svg width="12px" height="12px" viewBox="0 0 12 12" version="1.1">
```

Comment:
any way this could use the `icon-plus.svg` (like via an `img` tag or `background-img` rule) instead of duplicating its content here?

I always forget the best ways to use SVGs...I know it kinda depends on the situation.

If neither of those options works, I think we can actually use `import` to load the file. Let me know and I can help out doing it that way.

Reply:
Yeah I'd rather use it as a bg image - I'll see if I can work that out 👍